working/augmentations/feature-specification.md

```diff
@@ -1128,6 +1174,10 @@ and assume the third point is always true.
 
 ## Changelog
 
+### 1.38
+
+*   Generalize the treatment of default values of optional parameters.
```

Comment:
Can you add `(#4172)` here so that anyone reading this knows the issue this change fixes?